examples/atlas-advanced-cluster/multi-cloud/main.tf

```diff
@@ -0,0 +1,64 @@
+provider "mongodbatlas" {
+  public_key  = var.public_key
+  private_key = var.private_key
+}
+
+resource "mongodbatlas_advanced_cluster" "cluster" {
+  project_id     = mongodbatlas_project.project.id
+  name           = var.cluster_name
+  cluster_type   = "SHARDED"
+  backup_enabled = true
+
+  replication_specs {
+    num_shards = 3 # 3-shard Multi-Cloud Cluster
+
+    region_configs { # shard n1 
+      electable_specs {
+        instance_size = "M30"
+        node_count    = 3
+      }
+      analytics_specs {
+        instance_size = "M10"
+        node_count    = 1
+      }
+      provider_name = "AWS"
+      priority      = 7
+      region_name   = "US_EAST_1"
+    }
+
+    region_configs { # shard n2
+      electable_specs {
+        instance_size = "M30"
+        node_count    = 2
+      }
+      analytics_specs {
+        instance_size = "M10"
+        node_count    = 1
+      }
+      provider_name = "AZURE"
+      priority      = 6
+      region_name   = "US_EAST_2"
+    }
+
+    region_configs { # shard n3
```

Comment:
can we add an electrable_speccs for M10 cluster on GCP here as well (for consistency) instead of just GCP analytics_specs only?